doc/nrf/known_issues.rst

```diff
@@ -360,6 +360,19 @@ Subsystems
 Bluetooth LE
 ============
 
+.. rst-class:: v1-5-0 v1-4-2 v1-4-1 v1-4-0
+
+DRGN-15435: GATT Notifications and Write without Responses can be sent out of order.
+  GATT Notifications and Write without Responses can be sent out of order when not using a complete callback.
+
+  **Workaround:** Always set a callback for Notifications and Write without Response.
+
+.. rst-class:: v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0
+
+DRGN-15448: Incomplete bond overwrite during pairing procedure when peer is not using the IRK stored in the bond.
+  When pairing with a peer that has deleted its bond information and is using a new IRK to establish a new connection the existing bond will not be overwritten during the pairing procedure.
+  This can lead to MIC errors during reconnection if the old LTK is used instead.
```

Comment:
```suggestion
  When pairing with a peer that has deleted its bond information and is using a new IRK to establish the connection, the existing bond is not overwritten during the pairing procedure.
  This can lead to MIC errors during reconnection if the old LTK is used instead.
```